### ISSUE TITLE
[Question] What controllers are supported?

### ISSUE THREAD

**Issue by Toadfield** [kozec/sc-controller#616]
Is there a list for all supported controllers?
Cause I wanna use a switch pro controller and a ps3 controller.

**Comment by kozec** [comment]
Not really, assuming you are on Linux, SCC should work with anything kernel can handle. Both PS3 and switch controller should work.

**Comment by Toadfield** [comment]
thanks

**Comment by sp1ritCS** [comment]
@Toadfield are you able to use the switch pro controller? I was so far unable to get it working with sc-controller :/
(hid-nintendo, joycond)

It is detected by evtest:
```
Input driver version is 1.0.1
Input device ID: bus 0x3 vendor 0x57e product 0x2009 version 0x8111
Input device name: "Nintendo Switch Pro Controller"
[...]
```
but sc-controller is unable to pick it up

**Comment by Toadfield** [comment]
@sp1ritCS I didn't try,I have to wait to get my switch controller back,cause a friend borrowed it for some time.
